src/commands/statusline.ts

```
@@ -57,140 +130,274 @@ export const statuslineCommand = define({
+				// Process is dead, continue to update ourselves
+			}
+		}
 
-				return { blockInfo: 'No active block', burnRateInfo: '' };
-			}),
-			Result.inspectError(error => logger.error('Failed to load block data:', error)),
-			Result.unwrap({ blockInfo: 'No active block', burnRateInfo: '' }),
-		);
+		// Acquisition phase: Mark as updating
+		{
+			const currentPid = process.pid;
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)